fake-cucumber/javascript/src/FakeStepDefinitions.ts

```diff
@@ -0,0 +1,103 @@
+import { messages } from 'cucumber-messages'
+import {
+  ParameterTypeRegistry
+} from 'cucumber-expressions'
+import uuidv4 from 'uuid/v4'
+
+import Argument from 'cucumber-expressions/dist/src/Argument'
```

Comment:
Don't reach inside a package for imports. This should be:

 ```typescript
import { Argument } from 'cucumber-expressions'
```

Which will require a change to `cucumber-expressions`